Flashcards.AnaClos/Flashcards.AnaClos/Controllers/DataBaseController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace Flashcards.AnaClos.Controllers;
+
+public class DataBaseController
+{   
+    private string _connectionString;
+    private string _dataBaseName;
+    private string _server;
+
+    public DataBaseController()
+    {
+        _server = System.Configuration.ConfigurationManager.AppSettings.Get("Server");
+        _dataBaseName= System.Configuration.ConfigurationManager.AppSettings.Get("DataBase");
+        _connectionString = $@"Server={_server};Integrated Security=true;TrustServerCertificate=true";
```

Comment:
Conventionally, it is the connection string that is stored in the configuration file. Rather than the building blocks. The reason is easier, and also different SQL Installations (local/server/azure/version) require different connection string formats.

Reply:
I can see why you did it. Because you are creating the database and schema programmatically. Another solution, would be to provide an SQL script that creates the database and schema.